lib/katello/tasks/repository.rake

```diff
@@ -108,6 +114,18 @@ namespace :katello do
     end
   end
 
+  def handle_manifest_label_updates
+    batch_size = 1000
+    Katello::DockerManifest.where(schema_version: 2).find_in_batches(batch_size: batch_size) do |group|
```

Comment:
Perhaps we should filter out manifests with annotations, labels, or is_flatpak / is_bootable set to true, since users might have manifests that were synced after the change.

Reply:
This is just for the users who don't run the migration right away.